src/blister.c

```diff
@@ -2088,7 +1924,7 @@ Obj FuncSUBTR_BLIST (
 **
 *F  FuncMEET_BLIST( <self>, <list1>, <list2> ) . . . 
 **
-**  'FuncSUBTR_BLIST' implements the internal function 'MeetBlist'.
+**  'FuncMEET_BLIST' implements the internal function 'MeetBlist'.
```

Comment:
@stevelinton BTW you added this function MEET_BLIST in GAP 4.5.6 resp 4.6.2; but nothing uses it, other than some bugfix tests. What's the story there?